lib/generators/pakyow/app/templates/app/setup.rb

```diff
@@ -13,6 +13,7 @@
 
     # put global config here and they'll be available across environments
     app.name = '<%= app_name %>'
+    logger.enabled = true
```

Comment:
I tend to agree with @bryanp. The "right" solution feels like it'd be to make sure that the generated files point the correct things, but it doesn't feel good to change that file either (because it'd probably need to change again with the release). Not that we want to emulate Rails in any way, but they do this by supporting an `--edge` option to `new` which generates files pointing to github. I'm not sure it's worth all that effort at this point for such an edge case (pun!)

Seems like it'd be fair to write a separate issue and try to address it at some point.